test/WaveEquationCubedSphereTests.jl

```diff
@@ -7,6 +7,8 @@ using Plots
 using LinearAlgebra
 using WriteVTK
 using JLD
```

Comment:
Remove using JLD

Reply:
Updated. I've removed the `TypedTables`  for `DataFrames` as well